tests/integration/test_vulnerability_detector/test_scan_results/test_scan_different_cves.py

```diff
@@ -81,24 +134,49 @@ def mock_vulnerability_scan(request, mock_agent):
 @pytest.mark.skip(reason="It will be blocked by #1602, when it was solve we can enable again this test")
 def test_vulnerabilities_report(get_configuration, configure_environment, restart_modulesd, check_cve_db,
                                 mock_vulnerability_scan):
-    """This test checks what happens if a vulnerability is being reported by NVD
-    and not by the OS provider or vice-versa.
-
-    When a package has a specific provider defined, if this provider matches with the OVAL provider,
-    then the provider feed has priority versus the NVD one but, if the package's provider is a generic one,
-    then NVD feed has priority.
-
-    In this test, we check that NVD feed has priority with the generic package's provider and the
-    provider feed has priority for those packages where the provider matches the feeds' one.
-
-    Args:
-        get_configuration (fixture): Get configurations from the module.
-        configure_environment (fixture): Configure a custom environment for testing.
-        restart_modulesd (fixture): Reset the logs file and start a new monitor.
-        check_cve_db (fixture): Check if the CVE database exists and its tables are created.
-        mock_vulnerability_scan (fixture): It allows to mock the vulnerability scan inserting custom packages,
-                                           feeds and changing the host system.
-    """
+    '''
+    description: Check what happens if a vulnerability is being reported by NVD and not by the OS provider or
+                 vice-versa. To do this, it checks. When a package has a specific provider defined, if this provider
+                 matches with the OVAL provider, then the provider feed has priority versus the NVD one but, if the
```

Comment:
```suggestion
                 vice-versa. To do this, it checks when a package has a specific provider defined. If this provider
                 matches with the OVAL provider, then the provider feed has priority versus the NVD one but, if the
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/pull/2398/commits/c49756b4438164e5eed63e8ae690be47911eea23)